### PR TITLE
Refactored DELETE point according to REST

### PR DESCRIPTION
**Pull request by Todmy**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/garage-it/SmartHouse-backend/pull/25" target="_blank">ReviewNinja

File: src/API/scenarios/scenario.controller.js [hunk 1]
```diff
@@ -40,8 +40,9 @@ function update(req, res, next) {
 
 function remove(req, res, next) {
     req.scenario.removeAsync()
-        .then(() => {
-            res.end();
+        .then(sensor => {
+            res.json(sensor);
+            return next();
```
Comment (kosich) [inline src/API/scenarios/scenario.controller.js]: !fix
@Todmy @quasiunafantasia 
define the right behavior

Comment (quasiunafantasia) [inline src/API/scenarios/scenario.controller.js]: afaik, we don`t have any middlewares, which need to be executed after delete, so this code is equivalent for now

Comment (kosich) [inline src/API/scenarios/scenario.controller.js]: !fixed

**Pull request merged**